src/pages/login/login.css

```diff
@@ -0,0 +1,29 @@
+.form-login{
+    position: relative;
+    top : 5rem;
+    max-width: 400px;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+}
+.form-login input{
+    margin : 1rem;
+    padding: 0.5rem;
```

Comment:
Its better to use class name to target the `input` --> lower specificity  (unless u really need higher specifciity)